src/system_data.jl

```diff
@@ -711,6 +711,15 @@ function get_component(data::SystemData, uuid::Base.UUID)
     return nothing
 end
 
+function get_components(
+    filter_func::Function,
+    ::Type{T},
+    data::SystemData,
+) where {T}
+    return get_components(T, data.components, filter_func)
+end
+
+# Keep this version to support PowerSystems 1.x.
```

Comment:
I'm not sure that we should change IS to 2.0 just for this.

Reply:
I will leave the choice up to you. At the end of the day we will set compat in PSY 2.0 to a minimum version of IS.

Reply:
If we remove this function then we have to bump to 2.0. I say let's keep backwards compatibility.